android/wearable/src/main/java/me/tinykitten/trainlcd/MainActivity.kt

```
@@ -0,0 +1,161 @@
+        val intent = Intent(Intent.ACTION_VIEW)
+            .addCategory(Intent.CATEGORY_BROWSABLE)
+            .setData(Uri.parse(PLAY_STORE_URI))
+
+        lifecycleScope.launch {
+            try {
+                RemoteActivityHelper(applicationContext).startRemoteActivity(intent).await()
+
+                ConfirmationOverlay().showOn(this@MainActivity)
+            } catch (cancellationException: CancellationException) {
+                throw cancellationException
+            } catch (throwable: Throwable) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)